package.json

```diff
@@ -38,6 +38,7 @@
     "@szhsin/react-menu": "^4.0.0",
     "@tanstack/react-table": "^8.9.3",
     "@tremor/react": "^1.8.2",
+    "@types/google.accounts": "^0.0.14",
```

Comment:
Two things here:

* Types should be a dev dependency, not a dependency
* I don't think this type is actually needed? You're not adding any new libraries?